cl/users/templates/profile/favorites.html

```diff
@@ -44,6 +44,10 @@
         {% for type, typed_favorite_forms in favorite_forms.items %}
           <div class="tab-pane {% if forloop.first %}active{% endif %}" id="{{ type|slugify }}">
             {% if typed_favorite_forms %}
+            {% if type == "Dockets" %}
+            <a href="{{ docket_search_url }}" >Search Within Dockets</a>
+            {% endif %}
+
```

Comment:
We should definitely make this a button, but what about making it appear for all of the objects types? Seems not that hard? I also wonder about doing it via JS instead or perhaps building the link in the template (we have a `join` template tag, I think that might help). 

The other thought I had was that maybe this should be part of a header row with check boxes, similar to the gmail or github issues interfaces, where you can check the cases you want and then it'll give you a few actions you can do on each of them (delete, search, what else?). This would solve the problem that you might not want to search all of your dockets all the time. Instead of having one button, you check the ones you want and then you can search just those.

OTOH, I'm not sure how many useful features should go into this page. On the *other* other hand, I wonder about adding these features to the tags pages. I increasingly think they should just replace favorites and favorites should be allowed to languish a bit.

Reply:
Yeah, I'm not great with front-end beyond simple stuff, just wanted to get the backend code out there.

Reply:
The nice thing about doing it in the HTML is that it'd avoid having to send a docket_search_string, opinion_search_string, audio_search_string, etc., though doing that isn't...*horrible*. It's a simple page, so messing with that kind of stuff can be...OK, I guess. 

But I'd at least like to see buttons for all the object types equally before this lands. Is that possible? To make buttons in Bootstrap, it's this: https://getbootstrap.com/docs/3.3/css/#buttons

Reply:
Does searching audio make sense?

Reply:
Barely, but yes. You can search their titles and metadata and someday we hope to do speech to text to make the spoken words searchable.

Reply:
Everything should be in the latest version. I push 4 variables through and have 4 buttons.